packages/common/src/extensions/slickCellExternalCopyManager.ts

```diff
@@ -68,6 +69,26 @@ export class SlickCellExternalCopyManager {
         this._grid.focus();
       }
     });
+
+    if (typeof this._addonOptions?.onBeforePasteCell === 'function') {
+      const dataView = grid?.getData<SlickDataView>();
```

Comment:
we could probably check the `grid` object in the `if` condition above since I don't think it will run without it and it would be nearly impossible to be undefined anyway